packages/alea-frontend/pages/verify.tsx

```diff
@@ -0,0 +1,39 @@
+import { Typography } from '@mui/material';
+import { verifyEmail } from '@stex-react/api';
+import { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+import MainLayout from '../layouts/MainLayout';
+
+const VerifyEmail: NextPage = () => {
+  const [data, setData] = useState('');
+  const router = useRouter();
+
+  useEffect(() => {
+    if (!router.isReady) return;
+    async function fetchData() {
+      try {
+        const { query } = router;
+        const res = await verifyEmail(
+          query.email as string,
+          query.id as string
+        );
+        setData(res.data.message);
+      } catch (error) {
+        setData(error.response.data.message)
```

Comment:
`error` can be thrown from some other part of the code. so use `?` : `error?.response?.data?.message`